### ISSUE TITLE
Setting list layout to LV_LAYOUT_ROW_M does not create a horizontal list

### ISSUE THREAD

**Issue by simonemike95** [littlevgl/docs#94]
With the following code in place, my list does not function in the horizontal layout. I've tried numerous solutions found in other posts on issues here, with no luck. The only thing I can assume is that the LV_LAYOUT_ROW_M is broken/not working as intended for list objects.
![2019-12-19_08h04_00](https://user-images.githubusercontent.com/19980344/71175677-20acdf00-2236-11ea-8486-f913ef48ab9e.png)

My screen ends up looking like this:
![2019-12-19_08h06_09](https://user-images.githubusercontent.com/19980344/71175817-6a95c500-2236-11ea-9a31-bafe9dc909cd.png)

This is being tested on a simulated version, not running on my actual embedded device (currently not possible as it is highly custom, display drivers still WIP).

**Comment by kisvegabor** [comment]
Hi,

It was really broken. I've fixed it in `master`.

Note that the list buttons' width will be the same as list width before you change the layout. So the preferred order is:
1. `lv_obj_set_width(list, 100)`
2. Add buttons
3. `lv_list_set_layout(list, LV_LAYOUT_ROW_M);`
4. `lv_obj_set_width(list, 400)`

I've found another bug (still working on it) if you call `lv_list_set_layout` before adding buttons but the mentioned order should work.
